tests/frontend/README.md

```diff
@@ -1,14 +1,48 @@
 # e2e frontend tests
 
-Before you run tests create test files in malwarecage/tests/frontend/integration.
+**To create tests from cypress dev:**
+
+- Create test files in malwarecage/tests/frontend/integration.
+
+- Set configuration from terminal (project main directory):
+```
+source mwdb-vars.env
+export MALWARECAGE_ADMIN_LOGIN
+export MALWARECAGE_ADMIN_PASSWORD
+```
+- Run Malwarecage dev environment.
+- Set baseUrl in cypress.json to http://127.0.0.1
+
+- Then run from tests/frontend:
+```
+npm install cypress --save-dev
+```
+
+- To run tests in cypress dev environment :
+```
+./node_modules/.bin/cypress open
+```
+- To run tests as they run on docker and CI:
+```
+./node_modules/.bin/cypress run
+```
+- To run one test and leave browser open:
+```
+./node_modules/.bin/cypress run --no-exit --browser chrome --spec cypress/integration/sample.js
+```
+
 
 For more information how to write tests visit:
 https://docs.cypress.io/
 
-**To run tests:**
-
-Put your password in test files in malwarecage/tests/frontend/integration.
+**To run tests from docker:**
 
-Tests are included in e2e tests package and can be run from there.
+Set baseUrl in cypress.json to http://malwarefront
 
+Tests are included in e2e tests package and can be run from there:
+```
+sudo docker-compose -f docker-compose-e2e.yml down
+sudo docker-compose -f docker-compose-e2e.yml build web-tests
+sudo docker-compose -f docker-compose-e2e.yml up web-tests
```

Comment:
`sudo` is unnecessary if user is member of `docker` group. It shouldn't be added to instruction.
```suggestion
docker-compose -f docker-compose-e2e.yml down
docker-compose -f docker-compose-e2e.yml build web-tests
docker-compose -f docker-compose-e2e.yml up web-tests
```